[SRVCOM-1831] Serverless functions: update abstracts format (#46660)
openshift/openshift-docs: modules/serverless-testing-quarkus-functions.adoc

[id="serverless-testing-quarkus-functions_{context}"]
= Testing Quarkus functions

> You can test Quarkus functions locally on your computer. For that, run the Maven tests that are included in the project template.

Reviewer: ```suggestion
You can test Quarkus functions locally on your computer. Run the Maven tests that are included in the project template to test the functions.
```

Author: I'll leave this one as is. I think the transition in the beginning of the second sentence works better.

> You can test Quarkus functions locally on your computer. For that, run the Maven tests that are included in the project template.

Reviewer 2: For minimalism:

```suggestion
To test Quarkus functions locally on your computer, run the Maven tests that are included in the project template.
```
